BraintreeDataCollector/src/main/java/com/braintreepayments/api/PayPalDataCollector.java

```diff
@@ -65,7 +65,6 @@ String getClientMetadataId(Context context) {
      */
     @MainThread
     String getClientMetadataId(Context context, PayPalDataCollectorRequest request) {
-        // NEXT_MAJOR_VERSION In error scenarios, this function return null instead of "".
```

Comment:
We decided to take no action here since it's no longer necessary and it isn't a merchant-facing issue.